### PR TITLE
Raz/mgmt 1685 installcmd installation timeout

### PR DESCRIPTION
**Pull request by RazRegev**
New flag installation-timeout will be provided from env and
passed to installcmd. Represent timeout in minutes to wait for
control plane.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -31,6 +31,11 @@ OCM_CLIENT_SECRET := ${OCM_CLIENT_SECRET}
 ENABLE_AUTH := $(or ${ENABLE_AUTH},False)
 DELETE_PVC := $(or ${DELETE_PVC},False)
 
+ifneq ($(or $(INSTALLATION_TIMEOUT),),)
+        INSTALLATION_TIMEOUT_FLAG = --installation-timeout $(INSTALLATION_TIMEOUT)
+endif
```
Comment (eranco74) [inline Makefile:36]: Unsure why you need `($(or $(INSTALLATION_TIMEOUT),),)`?
```suggestion
ifdef INSTALLATION_TIMEOUT
        INSTALLATION_TIMEOUT_FLAG = --installation-timeout $(INSTALLATION_TIMEOUT)
endif
```

Comment (eranco74) [reply Makefile:36]: Consider moving it to the deploy section where it's used

File: internal/host/instructionmanager.go [hunk 2]
```diff
@@ -48,6 +49,7 @@ type InstructionConfig struct {
 	FreeAddressesImage      string `envconfig:"FREE_ADDRESSES_IMAGE" default:"quay.io/ocpmetal/assisted-installer-agent:latest"`
 	DhcpLeaseAllocatorImage string `envconfig:"DHCP_LEASE_ALLOCATOR_IMAGE" default:"quay.io/ocpmetal/assisted-installer-agent:latest"`
 	SkipCertVerification    bool   `envconfig:"SKIP_CERT_VERIFICATION" default:"false"`
+	InstallationTimeout     uint   `envconfig:"INSTALLATION_TIMEOUT" default:""`
```
Comment (eranco74) [inline internal/host/instructionmanager.go:52]: default should be 0 no?

Comment (RazRegev) [reply internal/host/instructionmanager.go:52]: this is equal to the zero value, it will initialize as 0

Comment (eranco74) [reply internal/host/instructionmanager.go:52]: I understand that empty string converts to but why initialize a uint with a string?

**Pull request merged**